controllers/operands/kubevirt_test.go

```diff
@@ -2059,7 +2059,8 @@ Version: 1.2.3`)
 							},
 						},
 					}
-					Expect(existingResource.Spec.Configuration.NetworkConfiguration.Binding[primaryUDNNetworkBindingName]).To(Equal(expectedInterfaceBindingPlugin))
+					Expect(existingResource.Spec.Configuration.NetworkConfiguration.Binding).To(HaveKeyWithValue(primaryUDNNetworkBindingName, primaryUserDefinedNetworkBinding()))
```

Comment:
Please consider instantiating an expected value for the `l2bridge`, similar to what was done for passt, as currently the unit test is not independent.

Reply:
imo it is fine like that, the function is simple and doesn't even gets parameters
it allows to change just one place and i don't see it reduce safety in a way that make this change required

Reply:
Another point is that is forces the reviewer of the test to look at the implementation details of `primaryUserDefinedNetworkBinding` in order to understand what is actually done.

Reply:
otoh it hides details, and allow him to look just if he desire to